### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by theo333**

### CODE REVIEW

**Review by alexwaite — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC"
+	"name": "grace-shopper-funky-monkey",
+	"version": "1.1.0",
+	"description": "",
+	"main": "db.js",
+	"dependencies": {
+		"babel-core": "^6.26.3",
```
Comment (leafoflegend) [inline package.json:7]: All `babel` and `webpack` dependencies are not `dependencies`, they are `devDependencies`.

Anything not used in your source code that clients or your server uses is not a dependency, it is a dependency for the developer to do work.

File: package.json [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
+	"devDependencies": {
+		"morgan": "^1.9.1",
+		"nodemon": "^1.18.11"
+	},
+	"scripts": {
+		"start": "node server/index.js",
+		"postinstall": "npm run webpack",
+		"webpack": "webpack",
+		"webpack:dev": "npm run webpack -- --mode=development --watch",
+		"start:dev": "npm run webpack:dev & DATABASE_URL=postgres://localhost/grace_shopper ./node_modules/.bin/nodemon server/index.js --ignore client/ --ignore public/"
```
Comment (leafoflegend) [inline package.json:32]: I prefer `./node_modules/.bin/` for everything, if you're going to do it one place, also do it for webpack.

File: server/db/index.js [hunk 3]
```diff
@@ -203,7 +202,7 @@ const syncAndSeed = () => {
 				]);
 			})
 			.then(([products, categories, users]) => {
-				console.log('categories: ', categories.map(cat => cat.get()));
+				// console.log('categories: ', categories.map(cat => cat.get()));
```
Comment (leafoflegend) [inline server/db/index.js:205]: Only leave comments if you also describe why a comment needs to be left.

**Review by leafoflegend — COMMENTED** [review]
I don't see any linting set up in your code base. Please set something up to keep linting consistent. Its as easy as picking a base you guys like. I think this might be a fine place to start: https://medium.com/@netczuk/your-last-eslint-config-9e35bace2f99

Any comment with `yarn` in that article can be replaced with `npm run`

File: package.json (hunk 1 above)
Comment (theo333) [reply package.json:7]: Fixed.

File: package.json (hunk 2 above)
Comment (theo333) [reply package.json:32]: Fixed.

**Pull request merged**